src/main/osd/osd_elements.c

```diff
@@ -992,6 +993,21 @@ static void osdElementFlymode(osdElementParms_t *element)
     }
 }
 
+static void osdElementReadyMode(osdElementParms_t *element)
+{
+    if (IS_RC_MODE_ACTIVE(BOXREADY) && !isFlying && calculateThrottlePercent() < 1) {
+            strcpy(element->buff, "READY"); 
+    }
+
+    if (IS_RC_MODE_ACTIVE(BOXREADY) && ARMING_FLAG(ARMED)) {
+        if (calculateThrottlePercent() > 1) {
+            isFlying = true;
+        }
+    } else { 
+        isFlying = false;
+    }
+}
```

Comment:
```suggestion
static void osdElementReadyMode(osdElementParms_t *element)
{
    static bool isFlying = false;

    if (IS_RC_MODE_ACTIVE(BOXREADY) && ARMING_FLAG(ARMED)) {
        if (calculateThrottlePercent() > 1) {
            isFlying = true;
        }
    } else { 
        isFlying = false;
    }

    strcpy(element->buff, isFlying ? "" : "READY");
}
```

Reply:
Need to remove the initialization on line 227 also

Reply:
Please see my comment, below.  We do not need to monitor throttle while disarmed, and we do not need the complexity of the `isFlying` boolean.

Reply:
@haslinghuis : `isFlying` is sticky when ARMed ..